projects/EIP-x stateless AA.MD

```diff
@@ -0,0 +1,143 @@
+Therefore we Propose a stateless Account abstraction model which doesn't force any change in the current Ethereum core but can solve the above problems at the account level. The advantage of having succint zk proof of the state is that it is possible to broadcast it across the nodes. Currently, to validate a block, light clients must request the state piece by piece, and this results in many light clients frequently burdening the full node altogether.
+
+
+## Project description
+
+The challenge at hand is to enable stateless verification of blocks in the blockchain system. The goal is to allow clients to verify the
+correctness of individual blocks without needing a full state. Instead, clients can rely on compact witness files generated by state-holding nodes, which contain the portion of the state accessed by the block along with proofs of correctness. This stateless verification offers several benefits, including supporting sharding setups and reducing reliance on validators. The state provider entity is at the core of stateless AA, providing a succinct and constant-sized proof of the current state. Light clients can effortlessly verify this proof against the mainnet, improving scalability, and reducing resource requirements for light clients. Users and applications can efficiently interact with the blockchain, relying on the security and integrity of zk-proofs to verify the state. This approach enables more participants to engage with the network without needing the computational resources and storage capacity required for full nodes.
+
+
+## Specification
+
+To sustain scalability, we propose Stateless Account Abstraction with Two new entities: 
+## 1- State provider: 
+this entity consists of Ethereum light clients  providing the latest  state, peer to peer network and  ZK circuit. 
+We introduce a decentralized peer-to-peer network with a state provider entity, collaborating to validate and generate ZK-SNARKs proofs 
+for state information.A lightweight client verifies proofs and validates state data from the state provider.
+
+A- ZK-SNARKs for State Proof:
+To provide a trustless and efficient proof of state, we employ ZK-SNARKs, a cryptographic primitive that allows a prover to generate 
+a succinct proof of the validity of a statement without revealing any sensitive information. By using ZK-SNARKs, the state provider can
+generate proofs of state integrity that can be verified by any participant in the peer-to-peer network, ensuring the trustworthiness of 
+the provided state.
+
+B- Peer-to-Peer Network:
+The proposed solution requires a decentralized peer-to-peer network to distribute the state provider functionality. Peers in the network 
```

Comment:
Does this use any existing p2p protocol? Is it overlay of Ethereum?

Reply:
I think it will be my job to implement a P2p protocol for consensus on state has been queried, still reading about Helios to have some sense. but next week its my major topic to go through it then i may request you for some help/brainstorms/finding someoen can advice on it please?